src/agent/func_services/func_node.js

```diff
@@ -31,8 +31,14 @@ class FuncNode {
     invoke_func(req) {
         return this._load_func_code(req)
             .then(func => new P((resolve, reject) => {
+                const max_heap_size_mb = Math.min(8, Math.ceil(func.config.memory_size / 1024 / 1024));
                 const proc = child_process.fork(FUNC_PROC_PATH, [], {
                         cwd: func.code_dir,
+                        execArgv: [
+                        process.execArgv,
```

Comment:
You want to use the spread operator to flatten the parent's args and append, like so:
```suggestion
                            ...process.execArgv,
```